bpf/kmesh/workload/xdp.c

```diff
@@ -186,15 +186,13 @@ int xdp_shutdown(struct xdp_md *ctx)
 
     // never failed
     parser_tuple(&info, &tuple_info);
-#ifdef AUTH_BY_XDP
```

Comment:
why remove

Reply:
use for debug， we need process both xdp-auth and user-mode-auth， before support all match action offerload to xdp